src/components/searchPanel/searchPanel.css

```diff
@@ -0,0 +1,30 @@
+.search-panel {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0 auto;
+    padding: 0 12px;
+    max-width: 1073px;
+    width: 100%;
+    height: 54px;
+    background: #FFF;
+    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+    border-radius: 8px;
+}
+
+.search-panel__input {
+    margin-left: 18px;
+}
+
+.search-panel__button {
+    display: block;
+    color: #FFF;
+    font-size: 12px;
+    font-style: normal;
```

Comment:
display: block;
font-style: normal;
are not needed. They are standard styles that do not need to be set.